pyproject.toml

```diff
@@ -56,11 +56,16 @@ all = [
   "catboost>=0.26; sys_platform!='darwin'",
   "catboost>=0.26,<1.2; sys_platform=='darwin'",
   "fastai",
+  "lightning",
   "mxnet",
+  "pytorch-ignite",
   "shap",
   "skorch",
   "tensorboard",
   "tensorflow",
+  "torch",
+  "torchaudio",
```

Comment:
Are `toachaudio` and `torchvision` actually required? (cf. [the doc](https://optuna.readthedocs.io/en/stable/reference/integration.html#dependencies-of-each-integration))